validation/genesis/validation-inputs/1946/meta.toml

```diff
@@ -0,0 +1,4 @@
+genesis_creation_commit = "4a487b8920daa9dc4b496d691d5f283f9bb659b1"
+node_version = "18.12.1"
+monorepo_build_command = "pnpm"
+genesis_creation_command = "opnode2"
```

Comment:
The `golang-validate-genesis-allocs` test that is failing attempts to reproduce the genesis.json from a commit of the `optimism` monorepo. It seems the commit hash above is not able to accurately reproduce the `genesis.json` provided in this pr. In order to get the test to pass locally, I made the following changes to this file:
```
genesis_creation_commit = "b047e1fd3b16d76c820d6e8cfc3cac4ede09d847"
node_version = "20.9.0"
monorepo_build_command = "pnpm"
genesis_creation_command = "forge1"
```
I also added these sepolia default values to the `deploy-config.json` otherwise the genesis creation script fails. Please ensure these values suit your chain:
```
    "faultGameClockExtension": 10800,
    "faultGameMaxClockDuration": 302400,
    "faultGameWithdrawalDelay": 604800,
```
If you agree with the changes, please push a commit to add them to your branch and we should see an all green CI test suite

Reply:
Thanks @bitwiseguy, that fixed genesis tests